src/btree/bt_vrfy_dsk.c

```diff
@@ -122,6 +122,8 @@ __wt_verify_dsk_image(WT_SESSION_IMPL *session, const char *tag, const WT_PAGE_H
         LF_CLR(WT_PAGE_ENCRYPTED);
     if (LF_ISSET(WT_PAGE_UNUSED))
         LF_CLR(WT_PAGE_UNUSED);
+    if (LF_ISSET(WT_PAGE_FT_UPDATE))
+        LF_CLR(WT_PAGE_FT_UPDATE);
     if (flags != 0)
         WT_RET_VRFY(session, "page at %s has invalid flags set: 0x%" PRIx8, tag, flags);
 
```

Comment:
Should probably have code in here to crosscheck the fast-delete information. (GH won't let me comment on the right place to put it, which is at line 390.) The timestamp is effectively a global stop time for the entire page, so it can be checked against the aggregate. I think. (Also, if there's still an underlying page, we probably ought to visit it and crosscheck that stop time against all the time windows on it; but that's probably best done as a follow-on ticket.)

Reply:
Good idea, I agree. Fixed in 1cd9a86, please take a look and confirm it's what you expected.

Reply:
What you added seems like a good thing to have, but it wasn't what I was thinking of, which was the timestamps. E.g., page_del->timestamp <= page_del->durable_timestamp; but also, ISTM at least, unpack->ta.newest_stop_durable_ts <= page_del->durable_timestamp, unpack->ta.newest_stop_ts <= page_del->timestamp, unpack->ta.newest_stop_txn <= page_del->txnid, and possibly also unpack->ta.prepare == 0 though that's less clear.

(That is: the delete timestamp is a stop time and is not included in the aggregate, so it should be newer than any/all stop times in the aggregate. Plus if there are prepared values on the page, they should have been resolved before the page got deleted, I think, but I'm not absolutely certain that's necessarily true.)

Reply:
We discussed this briefly and decided to defer these checks to a followup ticket. Then also decided to revert this change as all it really checks is whether the page flag test when unpacking the cell is actually there; if the page contents are wrong (that is, flag set without the extra elements being present, or vice versa) this check won't go off. But since that will desync the page scan from its contents, something else will fail, and that's probably good enough.